.travis.yml

```diff
@@ -1,34 +1,49 @@
-language: python
```

Comment:
I know this isn't the complete file but can you base what you have got on the latest structure in the `template-formula`?